lib/fungus_toast/games/games.ex

```diff
@@ -29,9 +31,9 @@ defmodule FungusToast.Games do
   Returns a list of games for a given user. The "active" parameter determines which games are returned
   """
   def list_games_for_user(%User{} = user),
-    do: list_games_for_user(user, ["Started", "Not Started"])
+    do: list_games_for_user(user, [Game.status_started, Game.status_not_started])
```

Comment:
To the idea of a `Game.Status` module, these could be pushed into a function on that module named `Game.Status.non_final_statuses` or something

Reply:
Also, re: the overloading on this function, I wonder of some of these deserve a different name altogether?  Like perhaps this could be `list_active_games_for_user(%User{} = user)`?

Reply:
Most of those list_games* functions were not used. I cleaned them up and created an active_statuses list per your suggestion. Thanks!